.github/workflows/release-scheduler.yml

```diff
@@ -1,32 +1,24 @@
 name: Release Scheduler
-
 on:
   schedule:
-    - cron: '15 15 * * MON' # Every Monday at 3:15pm UTC
+  - cron: '15 15 * * MON' # Every Monday at 3:15pm UTC
   workflow_dispatch:
-
-permissions:
-  contents: read
-
+permissions: read-all
 jobs:
   dispatch_scheduled_releases:
     name: Dispatch scheduled releases
-    if: ${{ github.repository == 'spring-projects/spring-security' }}
+    if: github.repository_owner == 'spring-projects'
     strategy:
       matrix:
-        #
         # List of active maintenance branches.
-        #
-        branch:
-          - 'main'
-          - '5.8.x'
-          - '5.7.x'
-          - '5.6.x'
+        branch: [ main, 5.8.x, 5.7.x, 5.6.x ]
     runs-on: ubuntu-latest
-    env:
-      TOKEN: ${{ secrets.GH_ACTIONS_REPO_TOKEN }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: spring-io/spring-gradle-build-action@v1
-      - name: Trigger release workflow
-        run: ./gradlew dispatchGitHubWorkflow -Pbranch=${{ matrix.branch }} -PworkflowId=update-scheduled-release-version.yml -PgitHubAccessToken=$TOKEN
+    - name: Checkout
```

Comment:
I'm noticing you are using a different indentation level for arrays. It doesn't seem to change the way the document is parsed, but I'm wondering if there is a standard for this or if it is just a preference? When I use IntelliJ's default yaml formatter, it indents arrays one level further than you have here.

Reply:
The standard practice is to not indent array items. That put the keys in an array item at the same level of indent as would the keys of a map. And that makes the YAML a lot easier to edit.